utils/rpc/src/mini_rpc_client.rs

```diff
@@ -0,0 +1,206 @@
+// TODO
+//  - WHEN UPGRADING TO 1.0.1 Client is in hyper-utils:
+//    Struct hyper_util::client::legacy::Client
+//  - use https for security reasons
```

Comment:
The Bitcoin Core RPC does not support https. You should connect on the same machine (or via a secure tunnel). There's work in progress to support unix domain sockets: https://github.com/bitcoin/bitcoin/pull/27375

Reply:
Thank you for pointing this out!